pkg/network/ovn_kubernetes_test.go

```diff
@@ -273,4 +301,5 @@ func TestOVNKubernetesIsSafe(t *testing.T) {
 	g.Expect(errs).To(HaveLen(2))
 	g.Expect(errs[0]).To(MatchError("cannot change ovn-kubernetes MTU"))
 	g.Expect(errs[1]).To(MatchError("cannot change ovn-kubernetes genevePort"))
+
```

Comment:
@JacobTanenbaum can you remove the extra newline?

Reply:
removed